app.py

```diff
@@ -17,12 +20,24 @@ def hello():
 
 
 @app.route("/predict", methods=["POST"])
-def predict():
```

Comment:
Could we have just one `predict` endpoint using POST, then delegate the logic to different handlers depending on content-type?

Reply:
It should be possible, though that sounds harder to use than doing different things depending on the method+params (GET vs POST).

Ideally I would just behave differently depending on the params instead, but that requires a bit more logic (doable though).

Is there any reason why you'd prefer using POST for both or differentiating by the content-type?